import-export-cli/cmd/importAPI.go

```diff
@@ -191,6 +217,16 @@ func mergeAPI(apiDirectory string, environmentParams *params.Environment) error
 	return nil
 }
 
+// isEndpointsFieldsValid returns false if either of the two fields: endpoints, loadBalanceEndpoints and failOverEndpoints are defined
+// in api_params.yaml file by the user mistakenly. This will return true , if only one of them is defined.
+func isEndpointsFieldsValid(endpoints *params.EndpointData, loadBalanceEndpoints *params.LoadBalanceEndpointsData, failoverEndpoints *params.FailoverEndpointsData) bool {
```

Comment:
Shall we break this to another line? maximum length is exceeded for this line.

Reply:
Fixed via https://github.com/wso2/product-apim-tooling/pull/596/commits/89238ea671a041105f709a8d49c1a3c6732c4e64